### PR TITLE
[DRAFT] Che docs contribution

### PR DESCRIPTION
**Pull request by deerskindoll**
blog post about contributing to che-docs

### CODE REVIEW

File: _posts/2024-08-01-contributing-to-che-docs.adoc [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+you can do it all in the environment you're already familiar with: an Eclipse Che workspace.
+
+.Getting started
+
+Use either link:https://eclipse.dev/che/docs/stable/administration-guide/installing-che/[your already installed Eclipse Che]
+or click the Developer Workspace button directly in the che-docs repository
+to directly open a clone of the repository in an Eclipse Che workspace hosted
+by Red Hat in the the link:https://developers.redhat.com/developer-sandbox?source=sso[Developer Sandbox].
+
+*add the button?*
```
Comment (ibuziuk) [inline _posts/2024-08-01-contributing-to-che-docs.adoc:27]: yes

File: _posts/2024-08-01-contributing-to-che-docs.adoc [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+*describe plugins?*
```
Comment (ibuziuk) [inline _posts/2024-08-01-contributing-to-che-docs.adoc:29]: yes

File: _posts/2024-08-01-contributing-to-che-docs.adoc [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+*vale sync command*
```
Comment (ibuziuk) [inline _posts/2024-08-01-contributing-to-che-docs.adoc:31]: yes

File: _posts/2024-08-01-contributing-to-che-docs.adoc [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+= Contributing to Eclipse Che documentation
+
+Writing about Eclipse Che is even easier than you think.
+The link:https://github.com/eclipse-che/che-docs[Eclipse Che documentation repository on GitHub] was created with community contribution in mind,
+featuring tools you need to create and test your content.
+And from the initial draft to submitting a PR,
+you can do it all in the environment you're already familiar with: an Eclipse Che workspace.
+
+.Getting started
```
Comment (ibuziuk) [inline _posts/2024-08-01-contributing-to-che-docs.adoc:20]: Please add `Devfile and How it Works?` section or smth. similar that would explain CDE, predefined commands etc.

File: _posts/2024-08-01-contributing-to-che-docs.adoc [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+*vale sync command*
+
+The Vale extension included in the che-docs repository follows the grammatical,
+spelling and stylistic rules from the link:https://redhat-documentation.github.io/supplementary-style-guide/[Red Hat supplementary style guide].
+Before you start writing,
+read the section about link:https://redhat-documentation.github.io/supplementary-style-guide/#technical-examples[documenting commands].
+
+.Writing tips
+
+A good technical document is easy to read and easy to follow.
```
Comment (ibuziuk) [inline _posts/2024-08-01-contributing-to-che-docs.adoc:40]:
```suggestion
A good technical document is easy to read, easy to follow, and now easy to contribute!
```

File: _posts/2024-08-01-contributing-to-che-docs.adoc [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+*describe plugins?*
+
+*vale sync command*
+
+The Vale extension included in the che-docs repository follows the grammatical,
+spelling and stylistic rules from the link:https://redhat-documentation.github.io/supplementary-style-guide/[Red Hat supplementary style guide].
+Before you start writing,
+read the section about link:https://redhat-documentation.github.io/supplementary-style-guide/#technical-examples[documenting commands].
+
+.Writing tips
```
Comment (ibuziuk) [inline _posts/2024-08-01-contributing-to-che-docs.adoc:38]: really like this section - a good way to wrap the blog up 👍

File: _posts/2024-08-01-contributing-to-che-docs.adoc (hunk 5 above)
Comment (ibuziuk) [reply _posts/2024-08-01-contributing-to-che-docs.adoc:40]: not ideal suggestion (feel free to skip it) but somewhere in the article we should articulate that the game-changer is contributing and spinning up an environment for contribution with one click

File: _posts/2024-08-01-contributing-to-che-docs.adoc [hunk 7]
```diff
@@ -0,0 +1,51 @@
+---
+title: Contributing to Eclipse Che documentation
```
Comment (ibuziuk) [inline _posts/2024-08-01-contributing-to-che-docs.adoc:2]: let's make the title catchy

Comment (ibuziuk) [inline _posts/2024-08-01-contributing-to-che-docs.adoc:2]:
```suggestion
title: Contributing to Eclipse Che documentation with one click
```

Comment (ibuziuk) [reply _posts/2024-08-01-contributing-to-che-docs.adoc:2]: not ideal, so feel free to come up with smth. better but as a reader, we should have smth. in the title that would grab the attention (the current one is very neutral for the blog imo)

File: _posts/2024-08-01-contributing-to-che-docs.adoc (hunk 4 above)
Comment (ibuziuk) [reply _posts/2024-08-01-contributing-to-che-docs.adoc:20]: image that might be useful here 
Screenshot 2024-07-26 at 12 27 07

File: _posts/2024-08-01-contributing-to-che-docs.adoc [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+featuring tools you need to create and test your content.
+And from the initial draft to submitting a PR,
+you can do it all in the environment you're already familiar with: an Eclipse Che workspace.
+
+.Getting started
+
+Use either link:https://eclipse.dev/che/docs/stable/administration-guide/installing-che/[your already installed Eclipse Che]
+or click the Developer Workspace button directly in the che-docs repository
+to directly open a clone of the repository in an Eclipse Che workspace hosted
+by Red Hat in the the link:https://developers.redhat.com/developer-sandbox?source=sso[Developer Sandbox].
```
Comment (dkwon17) [inline _posts/2024-08-01-contributing-to-che-docs.adoc:25]:
```suggestion
by Red Hat in the link:https://developers.redhat.com/developer-sandbox?source=sso[Developer Sandbox].
```

File: _posts/2024-08-01-contributing-to-che-docs.adoc [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+read the section about link:https://redhat-documentation.github.io/supplementary-style-guide/#technical-examples[documenting commands].
+
+.Writing tips
+
+A good technical document is easy to read and easy to follow.
+Here are a few tips that will help you write great technical content:
+
+* Keep it simple. Avoid self-referential language ("this article is about..."), overly long introductions, and deep-dives into the feature's history.
+* Keep it accessible. Short paragraphs, sentences and words make your content easy to read.
+* Keep the focus on the user. Write the content around what the user can achieve when they follow your instructions.
```
Comment (dkwon17) [inline _posts/2024-08-01-contributing-to-che-docs.adoc:45]: If it makes sense here, I think a bullet point about avoiding passive voice would be helpful

File: _posts/2024-08-01-contributing-to-che-docs.adoc [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+The link:https://github.com/eclipse-che/che-docs[Eclipse Che documentation repository on GitHub] was created with community contribution in mind,
+featuring tools you need to create and test your content.
+And from the initial draft to submitting a PR,
+you can do it all in the environment you're already familiar with: an Eclipse Che workspace.
+
+.Getting started
+
+Use either link:https://eclipse.dev/che/docs/stable/administration-guide/installing-che/[your already installed Eclipse Che]
+or click the Developer Workspace button directly in the che-docs repository
+to directly open a clone of the repository in an Eclipse Che workspace hosted
```
Comment (dkwon17) [inline _posts/2024-08-01-contributing-to-che-docs.adoc:24]:
```suggestion
to directly open a clone of the repository in an Eclipse Che CDE hosted
```

File: _posts/2024-08-01-contributing-to-che-docs.adoc [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+slug: /@deerskindoll/contributing-to-che-docs
+---
+
+= Contributing to Eclipse Che documentation
+
+Writing about Eclipse Che is even easier than you think.
+The link:https://github.com/eclipse-che/che-docs[Eclipse Che documentation repository on GitHub] was created with community contribution in mind,
+featuring tools you need to create and test your content.
+And from the initial draft to submitting a PR,
+you can do it all in the environment you're already familiar with: an Eclipse Che workspace.
```
Comment (dkwon17) [inline _posts/2024-08-01-contributing-to-che-docs.adoc:18]:
```suggestion
you can do it all in the environment you're already familiar with: an Eclipse Che cloud development environment (CDE).
```

File: _posts/2024-08-01-contributing-to-che-docs.adoc [hunk 12]
```diff
@@ -0,0 +1,51 @@
+---
+title: Contributing to Eclipse Che documentation
+layout: post
+author: Jana Vrbkova
+description: >-
+  Contribute to Eclipse Che documentation using an Eclipse Che workspace.
```
Comment (dkwon17) [inline _posts/2024-08-01-contributing-to-che-docs.adoc:6]:
```suggestion
  Contribute to Eclipse Che documentation using an Eclipse Che cloud development environment.
```

**Comment by ibuziuk** [comment]
@deerskindoll please, fix the build to make it work for preview - https://pr-check-59-che-blog.surge.sh/

File: _posts/2024-08-01-contributing-to-che-docs.adoc [hunk 13]
```diff
@@ -0,0 +1,106 @@
...
+
+Writing about Eclipse Che is even easier than you think.
+The link:https://github.com/eclipse-che/che-docs[Eclipse Che documentation repository on GitHub] was created with community contribution in mind,
+featuring tools you need to create and test your content.
+And from the initial draft to submitting a PR,
+you can do it all in the environment you're already familiar with: an Eclipse Che workspace.
+
+.Getting started
+
+Use either link:https://eclipse.dev/che/docs/stable/administration-guide/installing-che/[your own Eclipse Che installation]
```
Comment (ibuziuk) [inline _posts/2024-08-01-contributing-to-che-docs.adoc:22]: Let's mention it as a second option, or not mention it at all

File: _posts/2024-08-01-contributing-to-che-docs.adoc [hunk 14]
```diff
@@ -0,0 +1,106 @@
...
+or click the Developer Workspace button in the che-docs repository README file.
+This opens a clone of the repository in an Eclipse Che workspace hosted
+by Red Hat in the link:https://developers.redhat.com/developer-sandbox?source=sso[Developer Sandbox].
```
Comment (ibuziuk) [inline _posts/2024-08-01-contributing-to-che-docs.adoc:25]:
```suggestion
by Red Hat in the link:https://developers.redhat.com/developer-sandbox[Developer Sandbox].
```

File: _posts/2024-08-01-contributing-to-che-docs.adoc [hunk 15]
```diff
@@ -0,0 +1,106 @@
...
+Alternatively, you can click this button:
+
+[![Contribute](https://www.eclipse.org/che/contribute.svg)](https://workspaces.openshift.com/f?url=https://github.com/eclipse/che-docs)
```
Comment (ibuziuk) [inline _posts/2024-08-01-contributing-to-che-docs.adoc:28]:
```suggestion
[![Contribute](https://www.eclipse.org/che/contribute.svg)](https://workspaces.openshift.com#https://github.com/eclipse/che-docs)
```

File: _posts/2024-08-01-contributing-to-che-docs.adoc (hunk 13 above)
Comment (deerskindoll) [reply _posts/2024-08-01-contributing-to-che-docs.adoc:22]: got it

File: _posts/2024-08-01-contributing-to-che-docs.adoc (hunk 12 above)
Comment (deerskindoll) [reply _posts/2024-08-01-contributing-to-che-docs.adoc:6]: if I understand it correctly, it's better to refer to it as a CDE than a workspace (in general)? I want to make sure I use the correct expression in the future.

File: _posts/2024-08-01-contributing-to-che-docs.adoc (hunk 9 above)
Comment (deerskindoll) [reply _posts/2024-08-01-contributing-to-che-docs.adoc]: added

**Comment by deerskindoll** [comment]
> https://pr-check-59-che-blog.surge.sh/

@ibuziuk I don't think I can fix it on my end. the vale check fails because of:

`Error: Unable to resolve action `benoitf/vale-action@reviewdog`, unable to find version `reviewdog``

and the pull-request-info check fails because of:

`Error: The head commit for this pull_request event is not ahead of the base commit. Please submit an issue on this action's GitHub repo.`

**Comment by ibuziuk** [comment]
@deerskindoll it is probably related to the fact that PR is created from the fork

**Pull request closed without merging**